frontend/spec/features/products_spec.rb

```diff
@@ -234,57 +234,52 @@
-    within(:css, '#sidebar_products_search') { click_button 'Search' }
-
-    expect(page).to have_css('#products .product-list-item').exactly(3).times
-    tmp = page.all('#products .product-list-item a').map(&:text).flatten.compact
+  it 'is able to display products priced between 50 and 100 dollars' do
+    within(:css, '#collapseFilterPrice') { click_on '$50 - $100' }
+    expect(page).to have_css('.product-component-name').exactly(2).times
+    tmp = page.all('.product-component-name').map(&:text).flatten.compact
     tmp.delete('')
-    expect(tmp.sort!).to eq(['Ruby on Rails Mug', 'Ruby on Rails Stein', 'Ruby on Rails Tote'])
+    expect(tmp.sort!).to eq(['Ruby on Rails Mug', 'Ruby on Rails Tote'])
   end
 
-  it 'is able to display products priced between 15 and 18 dollars across multiple pages' do
-    Spree::Config.products_per_page = 2
-    within(:css, '#taxonomies') { click_link 'Ruby on Rails' }
-    check 'Price_Range_$15.00_-_$18.00'
-    within(:css, '#sidebar_products_search') { click_button 'Search' }
-
-    expect(page).to have_css('#products .product-list-item').twice
-    expect(page).to have_css('#products .product-list-item').twice
-
-    find('.pagination .next a').click
-    expect(page).to have_css('#products .product-list-item').once
+  it 'is able to display products priced between 101 and 150 dollars' do
+    within(:css, '#collapseFilterPrice') { click_on '$101 - $150' }
+    expect(page).to have_css('.product-component-name').once
+    tmp = page.all('.product-component-name').map(&:text).flatten.compact
+    tmp.delete('')
+    expect(tmp.sort!).to eq(['Ruby on Rails Bag'])
   end
 
-  it 'is able to display products priced 18 dollars and above' do
-    within(:css, '#taxonomies') { click_link 'Ruby on Rails' }
-    check 'Price_Range_$18.00_-_$20.00'
-    check 'Price_Range_$20.00_or_over'
-    within(:css, '#sidebar_products_search') { click_button 'Search' }
-
-    expect(page).to have_css('#products .product-list-item').exactly(4).times
-    tmp = page.all('#products .product-list-item a').map(&:text).flatten.compact
+  it 'is able to display products priced between 151 and 200 dollars' do
+    within(:css, '#collapseFilterPrice') { click_on '$151 - $200' }
+    expect(page).to have_css('.product-component-name').exactly(4).times
+    tmp = page.all('.product-component-name').map(&:text).flatten.compact
     tmp.delete('')
-    expect(tmp.sort!).to eq(['Ruby on Rails Bag',
-                             'Ruby on Rails Baseball Jersey',
+    expect(tmp.sort!).to eq(['Ruby on Rails Baseball Jersey',
                              'Ruby on Rails Jr. Spaghetti',
-                             'Ruby on Rails Ringer T-Shirt'])
+                             'Ruby on Rails Ringer T-Shirt',
+                             'Ruby on Rails Stein'])
+  end
+
+  it 'is able to display products priced between 151 and 200 dollars across multiple pages' do
+    Spree::Config.products_per_page = 3
```

Comment:
Doesn't it change the config for all specs running after this one? If yes, we must reset it to a default value after this scenario.

Reply:
It changes the config for only this one spec.